### ISSUE TITLE
WARNING:tensorflow:AutoGraph could not transform <function Model.make_train_function.<locals>.train_function at 0x0000021DB69A21F8> and will run it as-is.

### ISSUE THREAD

**Issue by lavish619** [tensorflow/tensorflow#41120]
```
import tensorflow as tf

model = tf.keras.models.Sequential([
    tf.keras.layers.Conv2D(16, (3,3), activation='relu', input_shape=(300, 300, 3)),
    tf.keras.layers.MaxPooling2D(2, 2),
    tf.keras.layers.Conv2D(32, (3,3), activation='relu'),
    tf.keras.layers.MaxPooling2D(2,2),
    tf.keras.layers.Conv2D(64, (3,3), activation='relu'),
    tf.keras.layers.MaxPooling2D(2,2),
    tf.keras.layers.Conv2D(64, (3,3), activation='relu'),
    tf.keras.layers.MaxPooling2D(2,2),
    tf.keras.layers.Conv2D(64, (3,3), activation='relu'),
    tf.keras.layers.MaxPooling2D(2,2),
    tf.keras.layers.Flatten(),
    tf.keras.layers.Dense(512, activation='relu'),
    tf.keras.layers.Dense(1, activation='sigmoid')
])

model.compile(loss='binary_crossentropy',
              optimizer="adam",
              metrics=['accuracy'])

from tensorflow.keras.preprocessing.image import ImageDataGenerator
train_datagen = ImageDataGenerator(rescale=1/255)

train_generator = train_datagen.flow_from_directory(
        r'C:\Users\asus\Downloads\tf horse human\\horsehuman', 
        target_size=(300, 300),  
        batch_size=128,
        class_mode='binary')

model.fit(train_generator,
          steps_per_epoch=8,  
          epochs=15)
```

Then it shows the warning.
WARNING:tensorflow:AutoGraph could not transform <function Model.make_train_function.<locals>.train_function at 0x0000021DB69A21F8> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: Bad argument number for Name: 4, expecting 3
To silence this warning, decorate the function with @tf.autograph.experimental.do_not_convert

8/8 [==============================] - 45s 6s/step - loss: 0.5747 - accuracy: 0.6699
Epoch 2/15
8/8 [==============================] - 47s 6s/step - loss: 0.2592 - accuracy: 0.8821
Epoch 3/15
8/8 [==============================] - 41s 5s/step - loss: 0.2060 - accuracy: 0.9121
Epoch 4/15
8/8 [==============================] - 40s 5s/step - loss: 0.1459 - accuracy: 0.9466
Epoch 5/15
8/8 [==============================] - 41s 5s/step - loss: 0.0600 - accuracy: 0.9800
Epoch 6/15
8/8 [==============================] - 40s 5s/step - loss: 0.0341 - accuracy: 0.9855
Epoch 7/15
8/8 [==============================] - 41s 5s/step - loss: 0.0334 - accuracy: 0.9911
Epoch 8/15
8/8 [==============================] - 40s 5s/step - loss: 0.0198 - accuracy: 0.9911
Epoch 9/15
8/8 [==============================] - 39s 5s/step - loss: 0.0085 - accuracy: 0.9989
Epoch 10/15
8/8 [==============================] - 44s 6s/step - loss: 0.0043 - accuracy: 0.9990
Epoch 11/15
8/8 [==============================] - 46s 6s/step - loss: 0.0025 - accuracy: 1.0000
Epoch 12/15
8/8 [==============================] - 40s 5s/step - loss: 0.0014 - accuracy: 1.0000
Epoch 13/15
8/8 [==============================] - 40s 5s/step - loss: 0.0014 - accuracy: 1.0000
Epoch 14/15
8/8 [==============================] - 40s 5s/step - loss: 5.8768e-04 - accuracy: 1.0000
Epoch 15/15
8/8 [==============================] - 40s 5s/step - loss: 7.8857e-04 - accuracy: 1.0000

**Comment by Saduf2019** [comment]
@lavish619 
Please refer to this comment and let us know.
I ran the code shared and face a different error, please find the [gist here](https://colab.research.google.com/gist/Saduf2019/589f516d5cc7ec1075cffcd889b9a24a/untitled260.ipynb), share all dependencies for us to replicate the issue faced.
#38947 link link2 #32377 #38691 #32319 #37251

**Comment by lavish619** [comment]
Yeah, this error was supposed to come because the dataset is stored on my device and I was using Jupyter Notebook instead of Google Colab, the directory written in code is a local directory on my laptop.

My current TensorFlow version is 2.2.
Python 3.7
gast 0.2.2(downgraded from 0.3.3) 
issue still persists.

Moreover this does not happen in google colab. No warning in colab.

> @lavish619
> Please refer to [this comment](https://github.com/tensorflow/tensorflow/issues/37144#issuecomment-600350256) and let us know.
> I ran the code shared and face a different error, please find the [gist here](https://colab.research.google.com/gist/Saduf2019/589f516d5cc7ec1075cffcd889b9a24a/untitled260.ipynb), share all dependencies for us to replicate the issue faced.
> #38947 link link2 #32377 #38691 #32319 #37251
> Also let us know the tf version.

**Comment by lavish619** [comment]
Somehow its automatically gone. I just restarted everything, installed anaconda and TensorFlow again, and checked correctly for paths. And now it's working fine. However, now it is showing different error on GPU but this warning is gone so closing this issue.

**Comment by qNOVAq** [comment]
For me running `pip install tensorflow` and  `pip install tensorflow-gpu` fixed the problem.
I already had it installed but few things were outdated and needed an update,  after the installation, I restarted Jupyter and everything is working just fine.
